test/unit/utils.spec.ts

```diff
@@ -27,198 +21,6 @@ describe("utils.ts", () => {
     })
   })
 
-  describe("amountOnVout", () => {
```

Comment:
Really interesting how we eliminate this entire set of extra logical steps previously needed

Reply:
its a combination of now using `tx-decoder` (which has unit tests) + having a simpler mapping of 1 utxo -> 1 ledger transaction instead of grouping utxos into 1 ledger transaction if they came from the same bitcoin transaction